schemas/com.snowplowanalytics.mobile/screen_summary/jsonschema/1-0-0

```diff
@@ -0,0 +1,104 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for an entity tracked with foreground/background/screen_end events with summary statistics about the screen view",
+  "self": {
+    "vendor": "com.snowplowanalytics.mobile",
+    "name": "screen_summary",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "foreground_sec": {
+      "type": "number",
+      "description": "Time in seconds spent on the current screen while the app was in foreground",
+      "minimum": 0,
+      "maximum": 2147483647
+    },
+    "background_sec": {
+      "type": [
+        "number",
+        "null"
+      ],
+      "description": "Time in seconds spent on the current screen while the app was in background",
+      "minimum": 0,
+      "maximum": 2147483647
+    },
+    "last_item_index": {
+      "type": [
+        "integer",
+        "null"
+      ],
+      "description": "Index of the last viewed item in the list on the screen",
+      "minimum": 0,
+      "maximum": 65535
+    },
+    "items_count": {
+      "type": [
+        "integer",
+        "null"
+      ],
+      "description": "Total number of items in the list on the screen",
+      "minimum": 0,
+      "maximum": 65535
+    },
+    "min_x_offset": {
+      "type": [
+        "integer",
+        "null"
+      ],
+      "description": "Minimum horizontal scroll offset on the scroll view in pixels",
+      "minimum": -2147483647,
+      "maximum": 2147483647
+    },
+    "max_x_offset": {
+      "type": [
+        "integer",
+        "null"
+      ],
+      "description": "Maximum horizontal scroll offset on the scroll view in pixels",
+      "minimum": -2147483647,
+      "maximum": 2147483647
```

Comment:
Can you get a negative x/y offset?

Reply:
Yeah, it can also get negative offset and often you get because the scroll view may not start at the top of the screen – the app may have some toolbar before it, so in order to reach offset 0, you need to scroll a bit.

Reply:
That was a poor explanation but basically you get a negative offset if there is a bit of a gap between the top of the scroll view and the start of the content and that happens under a few circumstances especially on iOS.